### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by Yassin362**
ID 27208336

### CODE REVIEW

**Comment by Yassin362** [comment]
ID 27208336

**Comment by Yassin362** [comment]
Thank you very much

File: res/values-ar/strings.xml [hunk 1]
```diff
@@ -1377,7 +1377,8 @@ Commands:
     <string name="You_are_already_friends_">أنتم بالفعل أصدقاء.</string>
     <string name="Account_must_be_1_day_old_to_send_mail_">يجب أن يكون عمر الحساب يومًا واحدًا لإرسال البريد.</string>
     <string name="Translator">متـرجـم</string>
-    <string name="SUCCESS">تم بنجاح</string>
+    <string name="SUCCESS"بنجاح 
+</string>
```
Comment (Jeii20) [inline res/values-ar/strings.xml:1381]: Please undo this mistake.

**Review by Jeii20 — CHANGES_REQUESTED** [review]

**Comment by Jeii20** [comment]
PR closed. Wrong change at line 1380 and 1381. Please be careful with the coding files.

**Pull request closed without merging**